src/components/Request/Pay/Views/GeneralViews/AlreadyPaid.view.tsx

```
@@ -7,21 +7,72 @@ import * as utils from '@/utils'
 import Link from 'next/link'
 
 export const AlreadyPaidLinkView = ({ requestLinkData }: { requestLinkData: _consts.IRequestLinkData | undefined }) => {
+
+    const chainName = consts.supportedPeanutChains && consts.supportedPeanutChains.find((chain) => chain.chainId == requestLinkData?.chainId)?.name
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)